core/devmode/src/main/java/io/quarkus/dev/RuntimeUpdatesProcessor.java

```diff
@@ -354,7 +353,7 @@ private String getFileExtension(File file) {
                         if (value > existing) {
                             ret.add(path);
                             log.infof("File change detected: %s", file);
-                            if (doCopy) {
+                            if (doCopy && !Files.isDirectory(file)) {
```

Comment:
It makes sense to add this condition for files, but it also makes me wondering if directories would need to be created. WDYT?

Reply:
A directory is only watched for changes. 

It will be created when coping its files resource files here - https://github.com/quarkusio/quarkus/blob/27381651e25fd30febf3017e04c9a73c96273aaf/core/devmode/src/main/java/io/quarkus/dev/RuntimeUpdatesProcessor.java#L323